SharedResources/en.lproj/Localizable.strings

```diff
@@ -101,6 +101,7 @@
 "fileProvider.error.biometricalAuthWrongPassword.message" = "The password that has been saved for %@ is wrong. Please try again and enter your password to re-enable %@.";
 "fileProvider.error.defaultLock.title" = "Unlock Required";
 "fileProvider.error.defaultLock.message" = "To access and show the contents of your vault, it has to be unlocked.";
+"fileprovider.error.reauthentication" = "Access to your vault \"%@\" was denied. Open the main app to check your connection and re-authenticate if needed.";
```

Comment:
```suggestion
"fileprovider.error.unauthorized.text" = "Access to your vault \"%@\" was denied. Open the main app to check your connection and re-authenticate if needed.";
```

Reply:
Refactored in commit https://github.com/cryptomator/ios/commit/0bfa725374156c1f5b041dc2202ab4598341f875.